### PR TITLE
MATIC to POL migration

### PR DESCRIPTION
**Pull request by hsutaiyu**

### CODE REVIEW

File: docs/pos/concepts/tokens/pol.md [hunk 1]
```diff
@@ -1,29 +1,28 @@
-!!! danger "Important update"
-
-    There is a proposal to transition the native token of the Polygon PoS network from MATIC to POL. This document will provide details on what this means for users of MATIC, the Polygon ecosystem, and the planned changes. Please read carefully and find more information [here](https://polygon.technology/blog/polygon-2-0-implementation-officially-begins-the-first-set-of-pips-polygon-improvement-proposals-released).
-
 POL is the native token upgrade for the Polygon ecosystem, designed for use in a wide range of activities and purposes, including as a tool for network participation and security. By staking POL, participants can actively contribute to the ecosystem as validators. Importantly, POL has many of the same features as MATIC, is built on OpenZeppelin's ERC20 implementations, and supports [EIP-2612](https://eips.ethereum.org/EIPS/eip-2612) for signature-based permit approvals.
```
Comment (EmpieichO) [inline docs/pos/concepts/tokens/pol.md]:
```suggestion
POL is the native token upgrade for the Polygon ecosystem, designed for use in a wide range of activities and various purposes. For instance, POL is used as a tool for network participation and security. By staking POL, participants can actively contribute to the ecosystem as validators. Importantly, like MATIC, POL is built on OpenZeppelin's ERC20 implementations, supports [EIP-2612](https://eips.ethereum.org/EIPS/eip-2612) for signature-based permit approvals, and thus inherits most of the features found in MATIC.
```

File: docs/pos/concepts/tokens/pol.md [hunk 2]
```diff
@@ -1,29 +1,28 @@
...
 
-Separately, you are encouraged to engage in the governance and decision-making processes put forward by the recent PIPs.
-Your involvement and input can play a significant role in shaping the future of the Polygon ecosystem.
+    In the case of POL, the initial amount (the total number of POL when the upgrade occurs) is 10 billion tokens — 1:1 with MATIC since this is an upgrade.
 
-You can participate in governance proposals to vote on various aspects of the Polygon ecosystem through the PIP program.
+## Do I need to do anything manually?
 
-### ...as a node delegator or operator?
+Please refer to the [MATIC to POL migration guide](../../get-started/matic-to-pol.md) for details on what action you need to perform, if any, depending on the chain they are currently located, i.e., Ethereum, Polygon PoS, Polygon zkEVM, etc.
```
Comment (EmpieichO) [inline docs/pos/concepts/tokens/pol.md]:
```suggestion
Please refer to the [MATIC to POL migration guide](../../get-started/matic-to-pol.md) for details on what action you need to take, if any, depending on the chain they are currently located, i.e., Ethereum, Polygon PoS, Polygon zkEVM, etc. 
```

File: docs/pos/concepts/tokens/pol.md [hunk 3]
```diff
@@ -1,29 +1,28 @@
...
+## Do I need to do anything manually?
 
-### ...as a node delegator or operator?
+Please refer to the [MATIC to POL migration guide](../../get-started/matic-to-pol.md) for details on what action you need to perform, if any, depending on the chain they are currently located, i.e., Ethereum, Polygon PoS, Polygon zkEVM, etc. 
 
-No. You can provide feedback on the proposed changes in the PIPs and monitor GitHub and the forum for new node software versions to remain compatible with your given chain when PIPs are approved by the community.
+!!! tip "Deep dive into POL"
 
-### ...as an application or tooling developer?
+    Read the detailed [blog post on the POL migration](https://polygon.technology/blog/save-the-date-matic-pol-migration-coming-september-4th-everything-you-need-to-know) to learn more about the POL token, its properties, and what it entails for the Polygon ecosystem.
```
Comment (EmpieichO) [inline docs/pos/concepts/tokens/pol.md]:
```suggestion
    Read the detailed [blog post on the POL migration](https://polygon.technology/blog/save-the-date-matic-pol-migration-coming-september-4th-everything-you-need-to-know) to learn more about the POL token, its properties, and what the migration means for the Polygon ecosystem.
```

Comment (EmpieichO) [reply docs/pos/concepts/tokens/pol.md]: It was not clear whether "it" refers to POL or migration. 
I didn't expect "entails" to be followed by "for the ... " 
I get the sense that the idea is to capture how the migration affects the ecosystem (without sounding like there's any negative effect).

**Review by EmpieichO — APPROVED** [review]

**Review by rahuldamodar94 — APPROVED** [review]

**Pull request merged**